OpenLIFULib/OpenLIFULib/transducer.py

```diff
@@ -70,6 +77,37 @@ def initialize_from_openlifu_transducer(
             SlicerOpenLIFUTransducerWrapper(transducer), model_node, transform_node
         )
 
+    def update_transform(self, transform_matrix:np.ndarray, transform_matrix_units:Optional[str]=None):
+        """ Update the transducer transform by postcomposing an additional matrix on top of the current transform.
+
+        The transform_matrix is assumed to be in "openlifu" style transducer coordinates, which is currently hardcoded to being LPS,
+        so this function does the needed conversions.
+
+        This function is useful for applying transform updates that come from algorithms in openlifu-python,
+        where the transform would be in openlifu conventions.
+        """
```

Comment:
@sadhana-r I just want to make you aware that this `SlicerOpenLIFUTransducer.update_transform` function is going to be part of my virtual fitting PR. I think your transducer tracking work would also involve an update to the transducer transform, so this could save you the effort of having to write that function.